src/main/java/org/folio/circulation/infrastructure/storage/ActualCostStorageRepository.java

```diff
@@ -0,0 +1,31 @@
+package org.folio.circulation.infrastructure.storage;
+
+import static org.folio.circulation.support.http.ResponseMapping.forwardOnFailure;
+
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.domain.ActualCostRecord;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.CollectionResourceClient;
+import org.folio.circulation.support.http.client.ResponseInterpreter;
+import org.folio.circulation.support.results.Result;
+
+public class ActualCostStorageRepository {
+  private final CollectionResourceClient actualCostRecordStorageClient;
+
+  public ActualCostStorageRepository(Clients clients) {
+    actualCostRecordStorageClient = clients.actualCostRecordStorageClient();
+  }
+
+  public CompletableFuture<Result<Void>> createActualCostRecord(
```

Comment:
create() methods in our repositories usually return CompletableFuture<Result<SomeContext>>, like 
 ```
public CompletableFuture<Result<LoanAndRelatedRecords>> createLoan(
    LoanAndRelatedRecords loanAndRelatedRecords) {
```
or
```
public CompletableFuture<Result<RequestAndRelatedRecords>> create(
    RequestAndRelatedRecords requestAndRelatedRecords) {
```
why is actual cost record different?

Reply:
The method is updated, thanks.